python/CUDAQuantumExtension.cpp

```diff
@@ -33,10 +34,11 @@
 #include "runtime/mlir/py_register_dialects.h"
 #include "utils/LinkedLibraryHolder.h"
 #include "utils/OpaqueArguments.h"
-#include "mlir/Target/LLVMIR/Dialect/LLVMIR/LLVMToLLVMIRTranslation.h"
 
-#include "cudaq/Optimizer/CAPI/Dialects.h"
+#include "../runtime/cudaq/platform/orca/orca_qpu.h"
```

Comment:
It looks like lines 21-34 are all `runtime/cudaq/...`. Can the leading `../` of line 38 be removed and then move the include up there?

Reply:
The header files on lines# 21-34 reside within the 'python' sub-directory, `<root>/python/runtime/cudaq` for example, https://github.com/NVIDIA/cuda-quantum/blob/main/python/runtime/cudaq/algorithms/py_draw.h. 
The new include comes from `<root>/runtime/cudaq`, i.e. https://github.com/NVIDIA/cuda-quantum/blob/main/runtime/cudaq/platform/orca/orca_qpu.h. Hence I put it at the bottom of the list.

Reply:
OK, thanks. I think the following path should work then: `#include "cudaq/platform/orca/orca_qpu.h"`

Reply:
Done. 8e6e94aca7e98ad31e16ba62286bb1d86c81cda9